File: Codigo/web/back-end/dto/forum_dto.py
class ForumDTO:

    def __init__(self, id, title, forumPosts, originalPost):
        self.id = id
        self.title = title
        self.forumPosts = forumPosts
        self.originalPost = originalPost

    def get_id(self):
        return self.id

    def set_id(self, id):
        self.id = id

    def get_title(self):
        return self.title

    def set_title(self, title):
        self.title = title

File: Codigo/mobile/lib/src/app/models/dto/forum/ForumDTO.dart
import 'package:inteligenciaindustrialapp/src/app/models/dto/forum/ForumPostDTO.dart';
import 'package:json_annotation/json_annotation.dart';

part 'ForumDTO.g.dart';

@JsonSerializable()
class ForumDTO {
  num id;
  String title;
  ForumPostDTO originalPost;
  List<ForumPostDTO> forumPosts;

  ForumDTO({this.id, this.title, this.originalPost, this.forumPosts});

  factory ForumDTO.fromJson(Map<String, dynamic> json) =>
      _$ForumDTOFromJson(json);

  Map<String, dynamic> toJson() => _$ForumDTOToJson(this);
}
